### DISCUSSION TITLE
Great documentation, one suggestion

### DISCUSSION THREAD

**Discussion by JohnBeals** [abraunegg/onedrive#1472] [General]
I really needed to get started with containers, and simultaneously had a problem: I lost access to OneDrive because my computer suddenly refuses to boot Windows (even ISO).  So, I got both in, so to speak, one container.

One suggestion though, showing how to make the container persistent through reboots, and also detached from the tty, might be helpful ie, ` run -dit --restart unless-stopped driveone/onedrive `

**Comment by abraunegg** [comment]
@JohnBeals 
PR's to update the documentation is always welcome. 

The `--restart unless-stop` is documented under Docker - see: https://github.com/abraunegg/onedrive/blob/master/docs/Docker.md#run-or-update-with-one-script - however it should also be used elsewhere within the document, or clearly articulated once you are happy with how the Docker container is running / operating, add that command switch in to keep it running.
